lib/lightspeed/request.rb

```diff
@@ -13,33 +15,32 @@ def self.verbose?
       !! @verbose
     end
 
-    def self.base_url
-      "https://api.merchantos.com/API"
+    def self.base_host
+      "api.merchantos.com"
+    end
+
+    def self.base_path
+      "/API"
     end
 
     def initialize(client, method:, path:, params: nil, body: nil)
+      @method = method
```

Comment:
I'd like us to standardise on whether we're using attr_accessors or not. I would rather that we did, but since it's your PR I'll leave it up to you.